commands/user_e2e_test.go

```diff
@@ -6,6 +6,8 @@ package commands
 import (
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/spf13/cobra"
+	// "github.com/spf13/pflag"
+	"log"
```

Comment:
You need to order the import differently to fix CI:
```suggestion
	"log"
	
	"github.com/mattermost/mattermost-server/v5/model"
	"github.com/spf13/cobra"
```